README.md

```diff
@@ -178,7 +179,7 @@ The vast majority of IT people have come to understand the advantages of moving
 app deployments from their own infrastructure to the public cloud, including 
 increased agility, world-class security, and reduced costs. Now, when they approach
 leadership with the idea of moving to the cloud, they will no longer have to deal with 
-fear that they might be empowering a competitor.
+fear that they might be perceived as empowering a competitor.
```

Comment:
Nope. It's not just perception, it's hard cold cash.  But I like all the other edits!

Reply:
Fair point.